decompiler/pipeline/dataflowanalysis/expressionpropagationfunctioncall.py

```diff
@@ -61,7 +62,16 @@ def _is_call_value_used_exactly_once(self, definition: Assignment) -> bool:
         False otherwise, or Call has more than one return value.
         """
         if len(return_values := definition.destination.requirements) == 1:
-            return len(self._use_map.get(return_values[0])) == 1
+            required_variable = return_values[0]
+            requiring_instructions = self._use_map.get(required_variable)
+
+            usages = 0
+            for instruction in requiring_instructions:
+                for variable in instruction.accept(RequiredVariablesVisitor()):
+                    if variable == required_variable:
+                        usages += 1
+
+            return usages == 1
```

Comment:
Could you please add a test for expression propagation call that tests exactly what being fixed by this pull request (i.e. call being not propagated twice when return value used more than once by the same instruction)?